roslibrust_gencpp/regenerate-test-msgs.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
I would prefer if this script was rust code instead of bash? Not a requirement, but I think not crazy?

Reply:
I'm going to leave the script since it is easier to run and demonstrates the CLI, but the rust test will also generate the source to compare against.